pybpodapi/bpod/emulator/emulator.py

```diff
@@ -76,6 +85,7 @@ def __init__(self, hardware):
         self._state = State(self.hardware)
         self._manual_override_events = queue.Queue()
         self._socket = None
+        self.emulator_mode = GUI_ENABLED
```

Comment:
I don't understand this. Why define an emulator mode attribute in the emulator class?

Reply:
Fixed.